packages/framer-motion/src/animation/utils/keyframes.ts

```diff
@@ -36,7 +36,14 @@ export function getKeyframes(
 
         if (isNone(keyframes[i])) {
             noneKeyframeIndexes.push(i)
-        } else if (typeof keyframes[i] === "string") {
+        }
+
+        // TODO: Clean this conditional, it works for now
+        if (
+            typeof keyframes[i] === "string" &&
+            keyframes[i] !== "none" &&
+            keyframes[i] !== "0"
```

Comment:
do we need to consider "0px" as well?

Reply:
no we want that to be used as a template